srcpkgs/qbe/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.1
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+short_desc="Small embeddable C compiler backend"
+maintainer="Bryce Vandegrift <bryce@brycevandegrift.xyz>"
+license="MIT"
+homepage="https://c9x.me/compile"
+distfiles="https://c9x.me/compile/release/qbe-${version}.tar.xz"
+checksum=7d0a53dd40df48072aae317e11ddde15d1a980673160e514e235b9ecaa1db12c
+
+case "$XBPS_TARGET_MACHINE" in
+	aarch64*) ;;
+	x86_64*) ;;
+# Don't run checks for non x86_64 or aarch64 machines
+	*) make_check=no;
+esac
```

Comment:
```suggestion
case "$XBPS_TARGET_MACHINE" in
	aarch64* | x86_64*) ;;
	# Don't run checks for non x86_64 or aarch64 machines
	*) make_check=no ;;
esac
```
But anyway, why is this done? Do the tests fail on other archs?

Reply:
Last time I checked, tests fail for i686. I haven't checked 32-bit arm, but I would assume that they might fail as well.